static/app/views/issueDetails/streamline/eventDetails.tsx

```diff
@@ -153,14 +159,16 @@ const GraphAlert = styled(Alert)`
   border: 1px solid ${p => p.theme.translucentBorder};
 `;
 
-const GroupContent = styled('div')<{navHeight?: number}>`
+const ExtraContent = styled('div')`
   border: 1px solid ${p => p.theme.translucentBorder};
   background: ${p => p.theme.background};
   border-radius: ${p => p.theme.borderRadius};
-  position: relative;
 `;
 
-const GroupContentPadding = styled('div')`
+const GroupContent = styled(ExtraContent)<{navHeight?: number}>`
```

Comment:
doesn't look used here
```suggestion
const GroupContent = styled(ExtraContent)`
```